quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -210,4 +229,8 @@ private File getPomFileFromMavenApplication(QuarkusCliRestService app) {
                 .findFirst()
                 .orElseThrow(() -> new RuntimeException("Malformed Maven Quarkus application. Missing pom.xml"));
     }
+
+    private String truncateCurrentJavaVersion() {
```

Comment:
The name of this method does not correspond with what it does.

And what is the point here? To use JDK which is not the default in the system?
I don't see why. CLI's `--java` supports `11` and `17` (`11` is default), so why not just try both options, regardless to the system default?